src/sentry/runner/initializer.py

```diff
@@ -1,7 +1,10 @@
+from __future__ import annotations
+
 import importlib.metadata
```

Comment:
this file has a _lot_ of `Any` -- mostly because the settings object is kinda a giant bag of haphazard global variables and not a well structured object :(